IntroOOP/AbstractFactory/src/com/globant/abstractfactory/factory/FactoryProducer.java

```diff
@@ -0,0 +1,13 @@
+package com.globant.abstractfactory.factory;
+
+public class FactoryProducer {
+
+  public static SQLFactory getFactory(String factoryType) {
+    if (factoryType.equalsIgnoreCase("MySQL")) {
```

Comment:
here you could get a null pointer exception when factoryType goes null, to avoid this, you could invert the check, "MySQL".equalsIgnoreCase(factoryType)

Reply:
Hello Sebastian. I have resolved this issue in the topic-introOOP branch adopting the yoda convention. It seems like I incorrectly merged early that branch into Master, that's why changes are not seen in this 'topic-Maven' branch. I'm sorry. Best Regards.